modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service for the most recent graph of update possibilities.
```

Comment:
nit: there are multiple update services, including Red Hat's hosted https://api.openshift.com/api/upgrades_info/graph and customer OpenShift Update Service deploys.  Can we say "an OpenShift Update Service" instead  of "the OpenShift Update Service" here?

Reply:
Also, Scott has been pushing for, and it makes sense to me, avoiding the word "graph" as far as possible.  The graph aspects of this are an internal implementation detail, and the relevant aspect for a cluster admin is that the update service provides information about update recommendations and risks that are specific to their cluster.